istioctl/cmd/proxystatus.go

```diff
@@ -44,9 +67,17 @@ func statusCommand() *cobra.Command {
 Retrieves last sent and last acknowledged xDS sync from Istiod to each Envoy in the mesh
 
 `,
-		Example: `  # Retrieve sync status for all Envoys in a mesh
+		Example: `  # Retrieve sync status of common XDS types for all Envoys in a mesh
```

Comment:
Do Istio users know what XDS is?  I propose we keep this line the same.  If we must change it, the capitalization used by Envoy is `xDS`, not "XDS".

Reply:
Restored original line.